Analysis/Tasks/PWGHF/HFCandidateCreatorCascade.cxx

```diff
@@ -0,0 +1,304 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file HFCandidateCreatorCascade.cxx
+/// \brief Reconstruction of heavy-flavour cascade decay candidates
+///
+
+#include "Framework/AnalysisTask.h"
+#include "DetectorsVertexing/DCAFitterN.h"
+#include "AnalysisDataModel/HFSecondaryVertex.h"
+#include "AnalysisCore/trackUtilities.h"
+#include "ReconstructionDataFormats/DCA.h"
+#include "ReconstructionDataFormats/V0.h"
+#include "AnalysisTasksUtils/UtilsDebugLcK0Sp.h"
+#include "AnalysisCore/HFSelectorCuts.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::aod::hf_cand_prong2;
+using namespace o2::analysis;
+
+void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+{
+  ConfigParamSpec optionDoMC{"doMC", VariantType::Bool, false, {"Perform MC matching."}};
+  workflowOptions.push_back(optionDoMC);
+}
+
+#include "Framework/runDataProcessing.h"
+
+//#define MY_DEBUG
+
+#ifdef MY_DEBUG
+using MyTracks = aod::BigTracksMC;
+#define MY_DEBUG_MSG(condition, cmd) \
+  if (condition) {                   \
+    cmd;                             \
+  }
+#else
+using MyTracks = aod::BigTracks;
+#define MY_DEBUG_MSG(condition, cmd)
+#endif
+
+/// Reconstruction of heavy-flavour cascade decay candidates
+struct HFCandidateCreatorCascade {
+
```

Comment:
Unnecessary blank line.

Reply:
I like it, is there a rule that says I cannot have it?